addons/web/views/webclient_templates.xml

```diff
@@ -355,9 +355,9 @@
             <t t-set="body_classname" t-value="'bg-100'"/>
 
             <div class="container mb-5 pb-5">
-                <div class="row justify-content-center no-gutters">
-                    <div class="col-sm-8 col-md-6 col-lg-4 o_database_list">
-                        <div t-attf-class="mt-3 mt-lg-5 pt-3 pt-lg-4 px-2 px-sm-4 pb-3 {{login_box_classes}}">
+                <div class="row justify-content-center">
+                    <div t-attf-class="card border-0 o_database_list w-100 mt-5 bg-100 {{login_card_classes}}" style="max-width: 300px">
```

Comment:
You made a mix of both my suggestions, you have to choose :p
- Either wrap the card in a `div.col-auto` (as children of rows must be columns)
- Or remove the row div and add `mx-auto` to center the card (I prefer this: less DOM and you do not want a row, you just want a centered unique component).

Also: in both cases the `w-100` class must be removed ;)